client/dashboard/sites/overview-plan-card/index.tsx

```diff
@@ -71,8 +114,10 @@ export default function PlanCard( { site }: { site: Site } ) {
 					<Stat
 						density="high"
 						strapline={ __( 'Bandwidth' ) }
-						metric="7.2 GB"
-						description="Unlimited"
+						metric={
+							bandwidth && site.is_wpcom_atomic ? filesize( bandwidth, { round: 1 } ) : 'Unlimited'
```

Comment:
As it's not a placeholder anymore, it's now missing translation 😄

Reply:
Whoops! 615c81f8bd939b4e2e63485b264527f9900156ad